src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,115 @@
+        String getQuery = "SELECT * FROM manufacturers WHERE id = " + id
+                + " AND is_deleted = false";
+        try (Connection connection = ConnectionUtil.getConnection();
+                     PreparedStatement getManufacturerStatement =
+                             connection.prepareStatement(getQuery)) {
+            ResultSet resultSet = getManufacturerStatement.executeQuery();
+            if (resultSet.next()) {
+                optionalManufacturer = Optional.of(createManufacturer(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get manufacturer with id = "
+                    + id + " from DB", e);
+        }
+        return optionalManufacturer;
+    }
+
+    @Override
+    public List<Manufacturer> getAll() {
+        List<Manufacturer> manufacturers = new ArrayList<>();
+        String getAllQuery = "SELECT * FROM manufacturers WHERE is_deleted = false";
+        try (Connection connection = ConnectionUtil.getConnection();
+                     PreparedStatement getAllManufacturersStatement =
+                                 connection.prepareStatement(getAllQuery)) {
+            ResultSet resultSet =
+                    getAllManufacturersStatement.executeQuery();
+            while (resultSet.next()) {
+                manufacturers.add(createManufacturer(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get all manufacturers from DB", e);
+        }
+        return manufacturers;
+    }
+
+    @Override
+    public Manufacturer update(Manufacturer manufacturer) {
+        String updateQuery = "UPDATE manufacturers SET name = ? , country = ? "
+                + "WHERE id = ? AND is_deleted = false";
+        try (Connection connection = ConnectionUtil.getConnection();
+                     PreparedStatement updateManufacturerStatement =
+                                connection.prepareStatement(updateQuery)) {
+            updateManufacturerStatement.setString(1,manufacturer.getName());
+            updateManufacturerStatement.setString(2,manufacturer.getCountry());
+            updateManufacturerStatement.setLong(3,manufacturer.getId());
+            return updateManufacturerStatement.executeUpdate() > 0
+                    ? manufacturer : new Manufacturer();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't update manufacturer : "
+                    + manufacturer + " in DB", e);
+        }
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String deleteQuery = "UPDATE manufacturers SET is_deleted = true WHERE id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                     PreparedStatement deleteManufacturerStatement = connection.prepareStatement(
+                             deleteQuery)) {
+            deleteManufacturerStatement.setLong(1,id);
```

Comment:
don't forget about space. Fix it everywhere 
```suggestion
            deleteManufacturerStatement.setLong(1, id);
```

Reply:
++